admin/modules.php

```diff
@@ -413,7 +450,7 @@
                 '<button type="submit" class="btn btn-danger" id="saveButton">' .
                     IMAGE_UPDATE .
                 '</button>&nbsp;' .
-                '<a href="' . zen_href_link(FILENAME_MODULES, 'set=' . $set . ($_GET['module'] != '' ? '&module=' . $_GET['module'] : ''), 'SSL') . '" class="btn btn-default" role="button" id="cancelButton">' .
+                '<a href="' . zen_href_link(FILENAME_MODULES, 'set=' . $set . ($_GET['module'] !== '' ? '&module=' . $_GET['module'] : ''), 'SSL') . '" class="btn btn-default" role="button" id="cancelButton">' .
```

Comment:
Should this be `!empty($_GET['module'])` ?

Reply:
Perhaps. I only touched that line to use strict comparison.

Reply:
There are, unfortunately, a slew of spots within `modules.php` where `$_GET['module']` is _presumed_ to be present.